Select/racer_test.go

```diff
@@ -0,0 +1,53 @@
+package racer
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+)
+
+func TestRacer(t *testing.T) {
+
+	t.Run("returns the fastest URL", func(t *testing.T) {
+		slowServer := makeDelayedServer(20 * time.Millisecond)
+		fastServer := makeDelayedServer(0 * time.Millisecond)
+
+		defer slowServer.Close()
+		defer fastServer.Close()
+
+		slowURL := slowServer.URL
+		fastURL := fastServer.URL
+
+		want := fastURL
+		got, err := Racer(slowURL, fastURL, 10*time.Second)
+
+		if err != nil {
+			t.Fatalf("did not expect an error but got one %v", err)
```

Comment:
it's better to use `%w` compared to `%v` for type Error. See https://stackoverflow.com/questions/61283248/format-errors-in-go-s-v-or-w

Reply:
ack